src/azure-cli/azure/cli/command_modules/profile/_help.py

```diff
@@ -36,11 +36,11 @@
     - name: Log in interactively.
       text: az login
     - name: Log in with username and password. This doesn't work with Microsoft accounts or accounts that have two-factor authentication enabled. Use -p=secret if the first character of the password is '-'.
-      text: az login -u johndoe@contoso.com -p VerySecret
-    - name: Log in with a service principal using client secret. Use -p=secret if the first character of the password is '-'.
-      text: az login --service-principal -u http://azure-cli-2016-08-05-14-31-15 -p VerySecret --tenant contoso.onmicrosoft.com
```

Comment:
Since https://github.com/Azure/azure-cli/pull/18312, `identifierUris` such as `http://azure-cli-2016-08-05-14-31-15` are no longer set on the service principal created by `az ad sp create-for-rbac`, so it cannot be used as `--username`. Only App ID works now.